src/literature/components/CollaborationList.jsx

```diff
@@ -1,51 +1,66 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { List } from 'immutable';
 
 import CollaborationLink from './CollaborationLink';
 import InlineList from '../../common/components/InlineList';
 
-class CollaborationList extends Component {
-  constructor(props) {
-    super(props);
+const REGEX_COLLABORATIONS_WITH_SUFFIX = /group|force|consortium|team/i;
```

Comment:
don't you want to anchor the regex at the end of the string with `$`?

Reply:
That's what I asked, but apparently it is not always at the end.